src/main/java/Duke.java

```diff
@@ -1,10 +1,225 @@
+                String userInputCompleteTaskString = userInput.nextLine();
+                int userInputCompleteTaskInt = Integer.parseInt(userInputCompleteTaskString);
+                items[userInputCompleteTaskInt-1].completeTask();
+                noOfTasksCompleted += 1;
+                userInput = new Scanner(System.in);
+                System.out.println("Enter an instruction: \nPlease select either todo / deadline / event / list / quit.");
+                userInputString = userInput.nextLine();
+            }
+        }
+        System.out.println("---------------------------------------");
+        System.out.println("You have " + itemCount + " tasks and completed " + noOfTasksCompleted + " in total. Congratulations! :>");
+        for (int i=1;i<itemCount+1;i++){
+            System.out.print(i + ".");
+            if (((items[i-1]).getClassName()).equals("todo")){
+                System.out.print("[T] ");
+            }
+            if (((items[i-1]).getClassName()).equals("deadline")){
+                System.out.print("[D] ");
+            }
+            if (((items[i-1]).getClassName()).equals("event")){
+                System.out.print("[E] ");
+            }
+            if (items[i-1].getTaskStatus() == false){
+                System.out.print("[✗] ");
+            }
+            else{
+                System.out.print("[✓] ");
+            }
+            System.out.print(items[i-1].getTaskName());
+            if (((items[i-1]).getClassName()).equals("todo")){
+                System.out.println(" ");
+            }
+            if (((items[i-1]).getClassName()).equals("deadline")||((items[i-1]).getClassName()).equals("event")){
+                System.out.println(" " + (items[i-1]).returnDate());
+            }
+
+        }
+        System.out.println("---------------------------------------\n\n");
+
+        System.out.println("---------------------------------------");
+        System.out.println("That's all boys. See you soon!");
+        System.out.println("---------------------------------------");
+
+
+        //Level 1
+        /*boolean echo = true;
+        while (echo){
+            Scanner userInput = new Scanner(System.in);
+            System.out.println("Enter a word: ");
+            String userInputString = userInput.nextLine();
+            if (userInputString.equals("bye")){
+                echo = false;
+                System.out.println("---------------------------------------");
+                System.out.println("What a shame :( See you again soon!");
+                System.out.println("---------------------------------------");
+            }
+            else{
+                System.out.println("---------------------------------------");
+                System.out.println("You typed " + userInputString);
+                System.out.println("---------------------------------------");
+            }
+        }*/
+    }
```

Comment:
The Duke class contains very long code. Would it be better to separate each class into different java files?